### ISSUE TITLE
Parameter validation requiring web access

### ISSUE THREAD

**Issue by Llaves** [tzapu/WiFiManager#795]
This [issue](https://github.com/tzapu/WiFiManager/issues/736) asks the question about validating user supplied parameters and explains how this can be done directly in the html without altering WiFiManager code. What about the case where the validation of the custom parameter requires accessing the web - e.g., validating a password? I don't see how this can be handled in the html, in part because the ESP still hasn't connected to the access point. Callback code can handle the validation, but how is the ESP switched back to act as an access point and html sent to the user device to inform the user of the validation failure?

**Comment by tablatronix** [comment]
It can be done in non ap mode, when accessing on network.

**Comment by Llaves** [comment]
Yes, I understand that in non-AP mode the code can access the web and validate the parameter. The part I'm unclear on is how, when returning to AP mode to access the web page sent to the user to inform them the parameter failed to validate and to re-enter the parameter. I expect others have faced this issue and I'm looking for an example or a suggestion of best practices for handling this use case.

**Comment by tablatronix** [comment]
Yeah like i said , The only option is to use webportal mode in sta mode, no way you are going to be switching back and forth , makes no sense. This is only available in development branch.
